scalability_and_performance/ztp-deploying-disconnected.adoc

```diff
@@ -51,98 +59,166 @@ include::modules/ztp-acm-adding-images-to-mirror-registry.adoc[leveloffset=+3]
+
+include::modules/ztp-stopping-the-existing-gitops-ztp-applications.adoc[leveloffset=+2]
+
+include::modules/ztp-topology-aware-lifecycle-operator.adoc[leveloffset=+2]
 
 .Additional resources
+* For information about Topology Aware Lifecycle Operator (TALO), see xref:../scalability_and_performance/cnf-talo-for-cluster-upgrades.html#cnf-about-topology-aware-lifecycle-operator-config_cnf-topology-aware-lifecycle-operator[About the Topology Aware Lifecycle Operator configuration].
+
+include::modules/ztp-required-changes-to-the-git-repository.adoc[leveloffset=+2]
 
-* For a complete `PtpConfig` CR that configures `linuxptp` as an ordinary clock, see xref:../networking/using-ptp.adoc#cnf-configuring-cvl-nic-as-ptp-slave_using-ptp[Configuring linuxptp services as an ordinary clock for Intel Columbiaville NIC].
+include::modules/ztp-installing-the-new-gitops-ztp-applications.adoc[leveloffset=+2]
 
-* For further information about configuring PTP fast event notifications, see xref:../networking/using-ptp.adoc#cnf-configuring-the-ptp-fast-event-publisher_using-ptp[Configuring PTP fast events].
+include::modules/ztp-roll-out-the-configuration-changes.adoc[leveloffset=+2]
+.Additional resources
+* For information about creating `ClusterGroupUpgrade` CRs, see xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#talo-create-cgu-cr_ztp-deploying-disconnected[Creating the ClusterGroupUpgrade CR].
 
-//../scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc#talo-apply-policies_cnf-topology-aware-lifecycle-operator[Applying update policies to managed clusters with the Topology Aware Lifecycle Operator]
 
-include::modules/ztp-creating-ztp-custom-resources-for-multiple-managed-clusters.adoc[leveloffset=+1]
+// Manual installation moved here
+
+include::modules/ztp-manually-install-a-single-managed-cluster.adoc[leveloffset=+1]
+
+include::modules/ztp-configuring-a-static-ip.adoc[leveloffset=+2]
+
+include::modules/ztp-discovery-iso.adoc[leveloffset=+2]
 
-include::modules/ztp-prerequisites-for-deploying-the-ztp-pipeline.adoc[leveloffset=+2]
+include::modules/ztp-checking-the-managed-cluster-status.adoc[leveloffset=+2]
 
-include::modules/ztp-installing-the-gitops-ztp-pipeline.adoc[leveloffset=+2]
+include::modules/ztp-configuring-the-cluster-for-a-disconnected-environment.adoc[leveloffset=+2]
 
-include::modules/ztp-preparing-the-ztp-git-repository.adoc[leveloffset=+3]
+include::modules/ztp-configuring-ipv6.adoc[leveloffset=+2]
 
-include::modules/ztp-preparing-the-hub-cluster-for-ztp.adoc[leveloffset=+3]
+include::modules/ztp-troubleshooting-the-managed-cluster.adoc[leveloffset=+2]
 
-include::modules/ztp-creating-the-site-secrets.adoc[leveloffset=+2]
+include::modules/ztp-applying-source-custom-resource-policies.adoc[leveloffset=+2]
 
-include::modules/ztp-creating-the-siteconfig-custom-resources.adoc[leveloffset=+2]
+include::modules/ztp-things-to-consider-when-creating-custom-resource-policies.adoc[leveloffset=+2]
```

Comment:
Creation of policies using PolicyGenTemplate is well covered in other portions of this document. I think these two sections should be dropped.